### ISSUE TITLE
Bug: Typo in playground ToastContainer transition prop 

### ISSUE THREAD

**Issue by Topman-14** [fkhadra/react-toastify#1062]
![scrnli_2_9_2024_2-31-37 PM](https://github.com/fkhadra/react-toastify/assets/98329531/deef5e52-47b9-4c77-8849-a809411126bb)

I think the image is pretty self-explanatory, In the playground, the transition prop in the ToastContainer is written as an Object property, I'd like to fix it, if its okay with you.

**Comment by AleksandrMaj** [comment]
Yeah, after I copied it in my application, I was a bit confused why I have there an error. It needs curly brackets to be accepted.

**Comment by fkhadra** [comment]
The documentation will get a refresh soon thanks for reporting.
